ql/termstructures/globalbootstrap.hpp

```diff
@@ -39,6 +39,8 @@ namespace QuantLib {
 template <class Curve> class GlobalBootstrap {
     typedef typename Curve::traits_type Traits;             // ZeroYield, Discount, ForwardRate
     typedef typename Curve::interpolator_type Interpolator; // Linear, LogLinear, ...
+    typedef std::function<Array(const std::vector<Time>&, const std::vector<Real>&)>
+        AdditionalErrors;
```

Comment:
Maybe `AdditionalPenalties` would be more clear?

Reply:
Sounds good. Would you like the member variable renamed as well, or just the type?

Reply:
I'd change the whole lot.

Reply:
Done. I also rebased this branch to make sure there are no conflicts with my previous PR.

Reply:
Thanks!